src/utils/useUpdateThrottled.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
```

Comment:
Perhaps we could move this to a customHooks folder (still inside utils) or something like that?